envs/monkey_zoo/blackbox/config_templates/performance.py

```diff
@@ -10,6 +10,7 @@ class Performance(ConfigTemplate):
             "3Q=(Ge(+&w]*",
             "`))jU7L(w}",
             "t67TC5ZDmz",
+            "Passw0rd!",
```

Comment:
Why this change?

Reply:
To test the performance of the powershell machines I think ....